tests/modules/individuals/resources/test_individual_crud.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+from app.modules.individuals.models import Individual
+import logging
+import json
+
+from tests.modules.individuals.resources import utils as individual_utils
+
+from tests import utils
+
+log = logging.getLogger(__name__)
+
+
+def test_create_read_individual(flask_app_client, researcher_1, encounter_1):
+
+    # TODO add checks for is_public
+    encounter_1.owner = researcher_1
+    response = individual_utils.create_individual(flask_app_client, researcher_1)
+
+    individual_guid = response.json['guid']
+
+    assert individual_guid is not None
+
+    read_individual = Individual.query.get(individual_guid)
+    read_individual.add_encounter(encounter_1)
+    assert read_individual is not None
+
+    response = individual_utils.read_individual(
+        flask_app_client, researcher_1, individual_guid
+    )
+    individual_guid = response.json['guid']
+    read_individual = Individual.query.get(individual_guid)
+    assert read_individual is not None
+
```

Comment:
you've created a delete_individual util but I don't think you're using it. That would resolve one of your codeCov errors if you did

Reply:
Yep, added to the create and read test now to complete the cycle.